### ISSUE TITLE
compute-val-loss doesn't exist !!

### ISSUE THREAD

**Issue by scstu** [fizyr/keras-retinanet#1002]
I am trying to compute the validation loss during training. However when excuting the following command with the  `--compute-val-loss` option:

`!keras_retinanet/bin/train.py --tensorboard-dir ./logs --snapshot-path '/content/drive/My Drive/snapshots' --image-min-side 480 --image-max-side 640 --compute-val-loss --weights snapshots/resnet50_coco_best_v2.1.0.h5 csv train_annotations_.csv labels.csv --val-annotations val_annotations_.csv`

I got the following Error !!
`

> Using TensorFlow backend.
> usage: train.py [-h]
>                 [--snapshot SNAPSHOT | --imagenet-weights | --weights WEIGHTS | --no-weights]
>                 [--backbone BACKBONE] [--batch-size BATCH_SIZE] [--gpu GPU]
>                 [--multi-gpu MULTI_GPU] [--multi-gpu-force] [--epochs EPOCHS]
>                 [--steps STEPS] [--lr LR] [--snapshot-path SNAPSHOT_PATH]
>                 [--tensorboard-dir TENSORBOARD_DIR] [--no-snapshots]
>                 [--no-evaluation] [--freeze-backbone] [--random-transform]
>                 [--image-min-side IMAGE_MIN_SIDE]
>                 [--image-max-side IMAGE_MAX_SIDE] [--config CONFIG]
>                 [--weighted-average] [--workers WORKERS]
>                 [--max-queue-size MAX_QUEUE_SIZE]
>                 {coco,pascal,kitti,oid,csv} ...
> train.py: error: unrecognized arguments: --compute-val-loss

any help to solve this issue
`

**Comment by de-vri-es** [comment]
Did you update to the latest master?

**Comment by scstu** [comment]
solved by updating to the latest master. 
thank you @de-vri-es

**Comment by mariaculman18** [comment]
I am working with the last version (0.5.1) and `--compute-val-loss` is either ignored or not recognize. Any clue what happened?

**Comment by mehboobillahi56** [comment]
@scstu Hi, can you help me how to update to the latest master?
